File: dbflow-tests/src/androidTest/java/com/raizlabs/android/dbflow/test/example/ColonyDatabase.java
package com.raizlabs.android.dbflow.test.example;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Description:
 */
@Database(name = ColonyDatabase.NAME, version = ColonyDatabase.VERSION)
public class ColonyDatabase {

    public static final String NAME = "Colonies";

    public static final int VERSION = 1;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/example/ColonyDatabase.kt
package com.raizlabs.android.dbflow.test.example

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Description:
 */
@Database(name = ColonyDatabase.NAME, version = ColonyDatabase.VERSION)
object ColonyDatabase {

    val NAME = "Colonies"

    val VERSION = 1
}
